packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -167,6 +179,40 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+export interface CompatVersion {
+	base: string;
+	delta: number;
+}
+
+export interface CompatVersionConfig {
+	name: string;
+	createWith: CompatVersion;
+	loadWith: CompatVersion;
+}
+
+export const getCrossVersionCompatConfig = (): CompatVersionConfig[] => {
+	const allDefaultDeltaVersions = defaultCompatVersions.currentVersionDeltas.map((delta) => ({
+		base: pkgVersion,
+		// TODO: temporary: using delta*10 to differentiate delta for major version instead of delta for internal version
+		delta: delta * 10,
+	}));
```

Comment:
This hack is because the getRequestedRange function doesn't understand how to span major versions. I've updated that function, though, so this hack should be removable.